app/src/main/kotlin/com/wire/android/ui/home/conversations/details/editguestaccess/Buttons.kt

```diff
@@ -46,7 +46,8 @@ fun CreateLinkButton(
         fillMaxWidth = true,
         onClick = onCreateLink,
         loading = isLoading,
-        state = if (isLoading || !isGuestAccessAllowed) WireButtonState.Disabled else WireButtonState.Default,
+        state = if (shouldDisableGenerateGuestLinkButton) WireButtonState.Disabled
```

Comment:
Shouldn't we keep accounting for `isLoading` as well?

Reply:
yes we are considering it in the viewModel, `isGeneratingGuestRoomLink`

```
fun shouldDisableGenerateGuestLinkButton() = !editGuestAccessState.isGuestRoomLinkFeatureEnabled ||
            !editGuestAccessState.isGuestAccessAllowed ||
            editGuestAccessState.isGeneratingGuestRoomLink
```

Reply:
Aha! Read it through and it went past me, sorry!